lib/guardian/post_guardian.rb

```diff
@@ -249,6 +249,18 @@ def can_delete_post_action?(post_action)
       !post_action.post&.topic&.archived?
   end
 
+  def can_notify_post?(post)
```

Comment:
Im a little bit mixed on this being in guardian, it's not really access checking... It's more business logic that we don't want to notify them in this case, but they could still see it if they wanted, they are admins...

Reply:
Thinking more about this, it would at least make more sense IMO if we would name it like this:

```
def can_receive_post_notification(post)
  # ...
end
```

We are checking if the user can receive notifications from a post, so `can_notify_post` is misleading IMO, as we were not checking if a user can notify a post.

I still think it's `post_alerter` logic and not guardian logic, but this way it would at least more precisely express a check against current user.

Reply:
I like it in a guardian... but the naming is hard ... I guess `can_receive_post_notifications` works ... will rename